CMakeLists.txt

```
@@ -598,16 +605,34 @@ else ( enable-midishare )
   unset ( MidiShare_LIBS CACHE )
 endif ( enable-midishare )
 
+unset ( OPENSLES_SUPPORT CACHE )
+unset ( OpenSLES_LIBS CACHE )
+if ( enable-opensles )
+  check_include_file ( SLES/OpenSLES.h OPENSLES_SUPPORT )
+  if ( OPENSLES_SUPPORT )
+    find_library ( OpenSLES_LIBS OpenSLES )
```

Sry about 89b96a616e08364efeb495c3feb37982d53ec4b4. I was concerned that opensles support only depends on whether it's headers were found. But if the library isn't found it would still be activated. Suggesting to change it as follows:

```suggestion
    find_library ( OpenSLES_LIBS OpenSLES )
    if ( NOT OpenSLES_LIBS )
      unset ( OPENSLES_SUPPORT )
    endif ( OpenSLES_LIBS )
```

---

indeed, thanks for the suggestion